src/main/java/htsjdk/tribble/readers/AsciiLineReaderIterator.java

```diff
@@ -75,7 +75,7 @@ public String peek() {
     private class TupleIterator extends AbstractIterator<Tuple<String, Long>> implements LocationAware {
         
         public TupleIterator() {
-            hasNext(); // Initialize the iterator, which appears to be a requirement of the parent class.  TODO: Really?
+            super.hasNext(); // Initialize the iterator, which appears to be a requirement of the parent class.  TODO: Really?
```

Comment:
Why add `super.`? This class doesn't implement `hasNext()` and it's `private`.

Reply:
It's to make it clear that it's `TupleIterator`'s superclass (`AbstractIterator`) that the `hasNext()` method resolves to, as opposed to `hasNext()` on the outer class, `AsciiLineReaderIterator`.

The default is the superclass, so this doesn't change the behaviour, it just makes it clear (and stops Spotbugs complaining). See https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#ia-potentially-ambiguous-invocation-of-either-an-inherited-or-outer-method-ia-ambiguous-invocation-of-inherited-or-outer-method